build.sbt

```diff
@@ -1,77 +1,193 @@
+                              .dependsOn(lib, macroLib % "compile-internal, test-internal")
+                              .settings(commonSettings, usesMacros)
+
+lazy val runtime1         = Project("daffodil-runtime1", file("daffodil-runtime1")).configs(TestDebug)
+                              .dependsOn(io, lib % "test->test")
+                              .settings(commonSettings)
+
+lazy val runtime1Unparser = Project("daffodil-runtime1-unparser", file("daffodil-runtime1-unparser")).configs(TestDebug)
+                              .dependsOn(runtime1, lib % "test->test", runtime1 % "test->test")
+                              .settings(commonSettings)
+
+lazy val core             = Project("daffodil-core", file("daffodil-core")).configs(TestDebug)
+                              .dependsOn(runtime1Unparser, lib % "test->test", runtime1 % "test->test")
+                              .settings(commonSettings)
+
+lazy val japi             = Project("daffodil-japi", file("daffodil-japi")).configs(TestDebug)
+                              .dependsOn(core)
+                              .settings(commonSettings)
+
+lazy val sapi             = Project("daffodil-sapi", file("daffodil-sapi")).configs(TestDebug)
+                              .dependsOn(core)
+                              .settings(commonSettings)
+
+lazy val tdml             = Project("daffodil-tdml", file("daffodil-tdml")).configs(TestDebug)
+                              .dependsOn(core, core % "test->test")
+                              .settings(commonSettings)
+
+lazy val cli              = Project("daffodil-cli", file("daffodil-cli")).configs(TestDebug)
+                              .dependsOn(tdml, sapi, japi) // causes sapi/japi to be pulled in to the helper zip/tar
+                              .settings(commonSettings, nopublish)
+                              .settings(libraryDependencies ++= Dependencies.cli) 
+
+lazy val test             = Project("daffodil-test", file("daffodil-test")).configs(TestDebug)
+                              .dependsOn(tdml, core % "test->test")
+                              .settings(commonSettings, nopublish)
+
+lazy val testIBM1         = Project("daffodil-test-ibm1", file("daffodil-test-ibm1")).configs(TestDebug)
+                              .dependsOn(tdml)
+                              .settings(commonSettings, nopublish)
+
+lazy val tutorials        = Project("daffodil-tutorials", file("tutorials")).configs(TestDebug)
+                              .dependsOn(tdml)
+                              .settings(commonSettings, nopublish)
+
+lazy val testStdLayout    = Project("daffodil-test-stdLayout", file("test-stdLayout")).configs(TestDebug)
+                              .dependsOn(tdml, core % "test->test")
+                              .settings(commonSettings, nopublish)
+
+
+
+lazy val commonSettings = Seq(
+  organization := "edu.illinois.ncsa",
+  version := "2.1.0-SNAPSHOT",
```

Comment:
This is annoying. We have to update version number in two places? Below in different build.sbt you access version.value. Can we do that here?

Reply:
I think this is the only place where a version needs to be updated. What's the second place you are seeing?

Reply:
Pilot error by me. I often am reading quickly, and see something in the pink section, forgetting that pink means deleted.... 
The side-by-side view is better, except the lines all wrap...

Disregard my comment, or just delete all of these.